language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -1801,6 +1802,29 @@ public static Boolean withinLockStatementNode(BallerinaCompletionContext context
         return qualifiers;
     }
 
+    /**
+     * Check whether the completion item type belongs to the types list passed.
+     *
+     * @param lsCItem   Completion item
+     * @param typesList List of types           
+     * @return {@link Boolean}
+     */
+    public static boolean isTypeCompletionItem(LSCompletionItem lsCItem, List<TypeDescKind> typesList) {
+
+        if (lsCItem.getType() != LSCompletionItem.CompletionItemType.SYMBOL) {
```

Comment:
We are ignoring `LSCompletionItem.CompletionItemType.TYPE` here. Is it intended? Or should we change the name of the method to something like `isOfType`

Reply:
+1. Changed the method name to `isCompletionItemOfType()`.